echion/cpython/tasks.h

```diff
@@ -199,10 +199,14 @@ inline PyObject* PyGen_yf(PyGenObject* gen, PyObject* frame_addr)
         if (frame.stacktop < 1 || frame.stacktop > (1 << 20))
             return NULL;
 
-        auto localsplus = std::make_unique<PyObject*[]>(frame.stacktop);
-        if (copy_generic(frame.localsplus, localsplus.get(), frame.stacktop * sizeof(PyObject*)))
-            return NULL;
-
+            auto localsplus = std::make_unique<PyObject*[]>(frame.stacktop);
+            // Calculate the remote address of localsplus array
+            auto remote_localsplus = reinterpret_cast<PyObject**>(reinterpret_cast<uintptr_t>(frame_addr) + offsetof(_PyInterpreterFrame, localsplus));
```

Comment:
Ah, of course! `frame.localsplus` is not a pointer to an array, it _is_ the array! Good catch!